packages/e2e-tests/plugins/interactive-blocks/directive-init/view.js

```diff
@@ -0,0 +1,69 @@
+( ( { wp } ) => {
+	const { store, directive, useContext, useMemo } = wp.interactivity;
+
+	// Mock `data-wp-show` directive to test when things are removed from the
+	// DOM.  Replace with `data-wp-show` when it's ready.
+	directive(
+		'show-mock',
+		( {
+			directives: {
+				'show-mock': { default: showMock },
+			},
+			element,
+			evaluate,
+			context,
+		} ) => {
+			const contextValue = useContext( context );
+			const children = useMemo(
+				() =>
+					element.type === 'template'
+						? element.props.templateChildren
+						: element,
+				[]
+			);
+			if ( ! evaluate( showMock, { context: contextValue } ) ) {
+				return null;
+			}
+			return children;
```

Comment:
That part is not required.

```suggestion
			if ( ! evaluate( showMock, { context: contextValue } ) ) {
				return null;
			}
			return element;
```